unique_toolkit/unique_toolkit/app/schemas.py

```diff
@@ -95,15 +97,31 @@ class ChatEventPayload(BaseModel):
     assistant_id: str
     user_message: ChatEventUserMessage
     assistant_message: ChatEventAssistantMessage
-    text: Optional[str] = None
-    additional_parameters: Optional[ChatEventAdditionalParameters] = None
-    user_metadata: Optional[dict[str, Any]] = None
-    tool_choices: Optional[list[str]] = Field(
+    text: str | None = None
+    additional_parameters: ChatEventAdditionalParameters | None = None
+    user_metadata: dict[str, Any] | None = Field(
+        default_factory=dict,
+    )
+    tool_choices: list[str] | None = Field(
         default=[],
```

Comment:
This is not part of the PR but would it make sense to make an empty list a default value? @klcd

Reply:
And if so, why even make it "optional". May be we can enforce the type to be a list?